### PR TITLE
eventcreate: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/eventcreate.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Create an event in a specific event log:
+
+`eventcreate /l {{log_name}} /t {{type}} /id {{id}} /d "{{message}}"`
+
+- Create an event with a specific source:
+
+`eventcreate /so {{source_name}} /t {{type}} /id {{id}} /d "{{message}}"`
+
+- Create an event for a remote machine:
```
Comment (sbrl) [inline pages/windows/eventcreate.md]: Is this in the local event log regarding a remote machine, or in a remote machine's event log?

Comment (pxgamer) [reply pages/windows/eventcreate.md]: It appears in the remote machines event log.

Comment (sbrl) [reply pages/windows/eventcreate.md]: Ah. Perhaps `Create an event in a remote machine's event log` would make it clearer?

File: pages/windows/eventcreate.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# eventcreate
+
+> Create custom entries in the event log.
+
+- Create a new event in the log:
+
+`eventcreate /t {{success|error|warning|information}} /id {{id}} /d "{{message}}"`
```
Comment (agnivade) [inline pages/windows/eventcreate.md]: Where does the id come from ?

Comment (pxgamer) [reply pages/windows/eventcreate.md]: It can be any number between 1 and 1000. You just choose it afaik.

Comment (agnivade) [reply pages/windows/eventcreate.md]: Ah, ok. So how about `Create a new event with a given id(1-1000) in the log` ?

**Pull request merged**